lib/error-handler.js

```diff
@@ -40,19 +48,20 @@ export default function errorHandler (errorResponse) {
       payloadData: config.data
     }
   }
-
   if (isObject(data)) {
     if ('requestId' in data) {
       errorData.requestId = data.requestId
     }
-    if ('sys' in data && 'type' in data.sys) {
+    if ('message' in data) {
       errorData.message = data.message
```

Comment:
Why not `errorData.message = data.message || ''` if no message is there we'll have an empty string which is fine, it will help to identify the errors with no messages coming from the API endpoint and maybe tell the backend to add something there